app/controllers/application_controller.rb

```diff
@@ -47,14 +47,33 @@ def deobfuscate(hash)
   end
 
   def show_notification_count
-    if current_user
-      @count = TaskManagement.
-               notifications_count(current_user.user_type, current_user.id)
-    end
+    @count = Notification.unseen(current_user).count if current_user
```

Comment:
This particular methods are used extensively in `task_managements` controller. I thought a need might arise for reusability. I'll move them to the `task_managements` controller.